elephant/test/test_phase_analysis.py

```diff
@@ -202,5 +202,131 @@ def test_regression_269(self):
         self.assertEqual(len(phases_noint[0]), 2)
 
 
+class PairwisePhaseConsistencyTestCase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):  # Note: using setUp makes the class call this
```

Comment:
Do you prefer `setUp` over `setUpClass`?

Reply:
They serve different use cases: `setUp` is run at each test while `setUpClass` is run only once at the test class initialization. Since you're initializing read-only values only once, `setUpClass` is preferred for the sake of runtime. If the initialization is fast, either way is fine.